### ISSUE TITLE
Accessibility - Blurry PDF Text

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor-Android#5]
Why is the text in the PDF along the left and bottom edge of each page blurry?

**Comment by aquilent-dgemoets** [comment]
The Android Sweat & Toil app uses a third-party integrated PDF reader.  The occasional blurriness is a limitation of the PDF reader.  That said, if time permits, we may be able to add a hook to ask the Android phone to open the PDF using software already installed on the phone.  [ezPDF Reader](https://play.google.com/store/apps/details?id=udk.android.reader) supports Text To Speech, so it's possible that sight-impaired user would have this PDF reader installed.  This is currently second on our "if time permits" list.

**Comment by aquilent-dgemoets** [comment]
ILAB has re-prioritized this issue and we will try to add a hook to ask the Android phone to open the PDF using software already installed on the phone.

**Comment by jfc3-dol** [comment]
PDF text is still blurry when switching from one page to another. Think you need to look into another PDF reader for this application, because I'm assuming it still has issues of not reading any of the text with TalkBack like before.

You should see what one @skeeJay for other DOL Android applications.

**Comment by aquilent-dgemoets** [comment]
To my knowledge, @skeeJay does not work on Android applications.  And none are listed on his GitHub profile page.

But as has been noted, the final version will have a hook to ask the Android phone to open the PDF using software already installed on the phone.  This is the Android way of solving a problem like this, and TalkBack users will no doubt have a TalkBack-compatible PDF viewer installed on their phone.

**Comment by skeeJay** [comment]
Correct, I've never built an Android application.

**Comment by jfc3-dol** [comment]
@skeeJay couldn't remember if you did or not. 

Do you know who did the OSHA Heat index Android application, since they would know?

**Comment by skeeJay** [comment]
It was done by the contractor ERG, whose contract has ended, so they are not available. But also, I don't think that app contains any PDFs, so no one associated with that app would necessarily have an answer.

**Comment by aquilent-dgemoets** [comment]
The latest beta (uploaded Friday 12/4) includes the ability to have the Android phone open the PDF using software already installed on the phone (such as one that reads PDFs via TalkBack).

**Comment by nmangu** [comment]
I am testing this now. The app opens the PDF using TalkBack. However, it is not reading the contents within the PDF. Is that a limitation? One possibility would be that the PDF was a scanned document and not an original written document in PDF format. Is there something called 'reading the scanned PDF's' in the scope?

**Comment by jfc3-dol** [comment]
PDF is not being read by Android.

**Comment by carolynhuang** [comment]
Cannot be resolved. As discussed, reading the PDFs are not something that can be fixed now because of software limitations on mobile or tablet devices to read them correctly.

**Comment by jfc3-dol** [comment]
I was hoping an update to TalkBack might make it easier to read the PDFs for visual users since they are blurry which makes them difficult to read.

**Comment by jfc3-dol** [comment]
Closing this issue because we created a new one to easily track any progress on finding a better way to display PDF or the content a different way.

Opened issue #67 in its place.
